src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import TodoWrapper from './components/TodoWrapperComponents/TodoWrapper';
+import Weather from './components/WeatherComponents/Weather';
+
+const App: React.FC = () => (
```

Comment:
you dont need to mention the type if it can be infered from the value,
fix this in all the components in the project